tests/cephadm/test_bootstrap.py

```diff
@@ -426,6 +544,11 @@ def verify_bootstrap(cls, args, response):
         validate_dashboard_user(cls, args.get("initial-dashboard-user"), response)
         validate_dashboard_passwd(cls, args.get("initial-dashboard-password"), response)
 
+    # Public Key and Private Key validations
```

Comment:
can we add both args `ssh-pub-key` and `ssh-private-key` while checking?

Reply:
Added both public key and private key conditions separately for validations